bin/pycbc_multi_inspiral

```diff
@@ -142,6 +142,39 @@ logging.basicConfig(format='%(asctime)s : %(message)s', level=log_level)
 inj_filter_rejector = pycbc.inject.InjFilterRejector.from_cli_multi_ifos(opt,opt.instruments)
 ctx = scheme.from_cli(opt)
 
+def network_chisq(chisq, chisq_dof, snr_dict):
+    ifos = sorted(snr_dict.keys())
+    chisq_per_dof = {ifo : chisq[ifo] / (2*chisq_dof[ifo] - 2) for ifo in ifos}
+    snr2 = {ifo : np.array(snr_dict[ifo]) *
+                np.array(snr_dict[ifo]).conj() for ifo in ifos}
+    coinc_snr2 = sum(snr2.values())
+    snr2_ratio = {ifo : snr2[ifo] / coinc_snr2 for ifo in ifos}
+    network_chisq = sum( np.array(chisq_per_dof.values()) *
+                    np.array(snr2_ratio.values()) )
```

Comment:
While `values()` should give things ordered in the same way, I wouldn't assume this. Explicitly loop over ifos (or do a list comprehension) and select chisq_per_dof[ifo] * snr2_ratio[ifo]